internal/admission/defaulting.go

```diff
@@ -123,8 +125,8 @@ func (w *DefaultingWebHook) InjectDecoder(decoder *admission.Decoder) error {
 	return nil
 }
 
-func labelPod(ctx context.Context, result validate.ValidationResult, pod *corev1.Pod) *corev1.Pod {
-	labelToApply := LabelForValidationResult(result)
+func labelPod(ctx context.Context, result validate.ValidationResult, pod *corev1.Pod, strictMode bool) *corev1.Pod {
```

Comment:
I wonder if this method should be under `DefaultingWebhook` to prevent the grown of number of parameters.

Reply:
We thought about it but in this case the only one information from `DefaultingWebhook` is `strictMode`.

Reply:
Okay, as it's private we can change it later.